clang/test/Driver/windows-compat-sycl-args.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -### --sycl -c %s 2>&1 | FileCheck %s
+// RUN: %clang -### -fsycl-device-only -fsycl %s 2>&1 | FileCheck %s
+// CHECK: -fdelayed-template-parsing
+// CHECK: -fms-compatibility
+// REQUIRES: system-windows
+// expected-no-diagnostics
+
```

Comment:
Please put more details of what specific compatibility changes you doing and why in the commit message.